src/Uno.UWP/UI/ViewManagement/StatusBar/StatusBar.iOS.cs

```diff
@@ -24,6 +24,11 @@ public sealed partial class StatusBar
 	{
 		partial void InitializePartial()
 		{
+			NSNotificationCenter.DefaultCenter.AddObserver(
```

Comment:
The PR title says "ios skia," but if I'm not mistaken, this change (due to the file) only affects the Native iOS implementation. 

@MartinZikmund  could you please confirm?